source/frontend/src/components/maps/leaflet/Layers/PointClusterer.tsx

```diff
@@ -100,38 +101,23 @@ export const PointClusterer: React.FC<React.PropsWithChildren<PointClustererProp
 
   const pimsLocationFeatures: FeatureCollection<Geometry, PIMS_Property_Location_View> =
     useMemo(() => {
-      if (mapMachine.isFiltering && mapMachine.mapFeatureData.pimsLocationFeatures !== null) {
```

Comment:
don't need this if/else anymore because the list of activePimsPropertyIds is always active now.